projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html

```diff
@@ -1,32 +1,39 @@
-    <h2 class="govuk-accordion__section-heading">
-      <button type="button" id="accordion-with-summary-sections-heading-1"
+<div id="govuk-accordion__section-{{ case.caseId }}" class="govuk-accordion__section">
+  <div class="govuk-grid-row govuk-case-header">
+    <div class="govuk-grid-column-three-quarters">
+      <h3 id="case-title" class="govuk-case-title">{{ case.caseTitle }}</h3>
+      <h1 id="case-id" class="govuk-case-sub-title">{{ case.caseId }}</h1>
+    </div>
+    <div class="govuk-grid-column-twenty-percent">
+      <button id="btn-deselect-case" class="govuk-button hmcts-button--secondary" (click) = "onDeselect(case)">Deselect case</button>
+    </div>
+    <div class="govuk-accordion__section-header govuk-grid-column-five-percent">
+      <div class="govuk-accordion__section-heading">
+        <button type="button" id="accordion-with-summary-sections-heading-1"
               aria-controls="accordion-with-summary-sections-content-1" class="govuk-accordion__section-button"
               aria-describedby="accordion-with-summary-sections-summary-1" aria-expanded="false">
-        {{ case.caseTitle }}
         <span class="govuk-accordion__icon" aria-hidden="true"></span></button>
-    </h2>
-    <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-1">
-      {{ case.caseId }}
+      </div>
     </div>
   </div>
   <div id="accordion-with-summary-sections-content-1" class="govuk-accordion__section-content"
        aria-labelledby="accordion-with-summary-sections-heading-1">
-    <p class="govuk-body govuk-!-font-size-19">All users with access to this case.</p>
+    <div class="govuk-grid-row">
+      <span id="access-info" class="govuk-div-align-left">All users with access to this case.</span>
+    </div>
     <table class="govuk-table">
       <thead class="govuk-table__head">
       <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="col">Name</th>
-        <th class="govuk-table__header" scope="col">Email address</th>
-        <th class="govuk-table__header" scope="col">Actions</th>
+        <th id="th-name" class="govuk-table__header" scope="col">Name</th>
+        <th id="th-email" class="govuk-table__header" scope="col">Email address</th>
+        <th id="th-action" class="govuk-table__header" scope="col">Actions</th>
       </tr>
       </thead>
       <tbody class="govuk-table__body">
-      <tr class="govuk-table__row" *ngFor="let user of case.sharedWith; trackBy: trackByUserId">
-        <td class="govuk-table__cell">{{ user.firstName + ' ' + user.lastName }}</td>
-        <td class="govuk-table__cell">{{ user.email }}</td>
-        <td class="govuk-table__cell"><a href="#"></a></td>
+      <tr class="govuk-table__row" *ngFor="let user of case.sharedWith; index as idx; trackBy: trackByUserId">
+        <td id="user-full-name-{{ idx }}" class="govuk-table__cell">{{ user.firstName + ' ' + user.lastName }}</td>
```

Comment:
@rayliangatos  I suggest we dont need this id attribute. You could do it with class.

Reply:
class can be duplicated, and it's not resilient, use id is a better practice.

Reply:
Agree to disagree